Example/Tests/Connection/MHVSodaConnectionTests.m

```diff
@@ -1098,7 +1100,7 @@
                 
                 it(@"should forward the error to the caller", ^
                    {
-                       [[expectFutureValue(expectedError) shouldEventually] beNonNil];
+                       [[expectFutureValue(expectedError) shouldEventuallyBeforeTimingOutAfter(3)] beNonNil];
```

Comment:
What happened here? Why is it now taking 3 seconds to complete? I'm not excited about adding delays to our tests because they will take longer to complete.